core/thread.go

```diff
@@ -141,6 +141,16 @@ func (t *Thread) Head() (string, error) {
 	return mod.Head, nil
 }
 
+// LatestFiles returns the most recent files block
+func (t *Thread) LatestFiles() *pb.Block {
```

Comment:
This new method is used to get the latest files block (avatar) from the account thread, but may have other uses in the future.

Reply:
yeah, nice